src/Pages/Layout/Layout.js

```diff
@@ -0,0 +1,98 @@
+import React, { PureComponent } from 'react';
+import ReactGA from 'react-ga';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import AsyncComponent from '../Utils/AsyncComponent';
+import Header from './Header/Header';
+import Isna from './Isna/Isna';
+import * as styles from './Layout.module.css';
+
+class Layout extends PureComponent {
+  render() {
+    return (
+      <BrowserRouter>
+        <div
+          className={`${styles.bgGrayAlternative} ${styles.paddingTop68} ${
+            styles.stretch
+          } black-alternative lh-title sans-serif`}
+        >
+          <Header />
+          <Route
+            component={() => {
+              /*
+                Return the scroll to the top of the page when navigating
+                https://github.com/ReactTraining/react-router/issues/2019#issuecomment-292711226
+              */
+              window.scroll({
+                behavior: 'smooth',
+                left: 0,
+                top: 0
+              });
+
+              if (process.env.NODE_ENV === 'production') {
+                ReactGA.pageview(window.location.href);
+              }
+
+              return null;
+            }}
+          />
+          <Switch>
+            <Route
+              exact
+              path="/"
+              component={AsyncComponent({
```

Comment:
All components should be extracted from render as they are not using anything from the closure